### DISCUSSION TITLE
is the new update for the modrinth launcher a malware? (the version 0.7.1)

### DISCUSSION THREAD

**Discussion by BloodSpiritGod** [modrinth/.github#240] [Other discussions]
my anti-malware software quarantined and deleted the modrinth.exe the moment i tryed to update the launcher. 
![example](https://github.com/modrinth/.github/assets/163886974/b4732011-1af9-4815-b4dd-99465e853182)

**Comment by caela-uk** [comment]
It is a false alarm/flag.
Also if you think something is malware dont ask the creators or supporters if it is, ask different communities and users lmao.
